gspread/models.py

```diff
@@ -1339,6 +1339,43 @@ def insert_row(
 
         return data
 
+    def insert_rows(self, values, index=1, value_input_option='RAW'):
+        """Adds multiple rows to the worksheet at the specified index
+        and populates it with values.
+        The input should be a list of lists, with the lists each
+        containing one row's values.
+        Widens the worksheet if there are more values than columns.
+        :param values: List of row lists.
+        """
+
+        body = {
+            "requests": [{
+                "insertDimension": {
+                    "range": {
+                      "sheetId": self.id,
+                      "dimension": "ROWS",
+                      "startIndex": index - 1,
```

Comment:
It looks like there's a problem with insert indices.

Steps to reproduce:

1. I have a sheet with consecutive values:

![Screenshot 2020-04-27 at 15 00 55](https://user-images.githubusercontent.com/264674/80375257-48711b00-8898-11ea-9891-f4b83839decb.png)

2. I call 

```python
wks.insert_rows([['r1.1','r1.2'],['r2.1', 'r2.2']], index=3)
```

Where `wks` is my worksheet.

3. I get 

![Screenshot 2020-04-27 at 15 01 20](https://user-images.githubusercontent.com/264674/80375364-78b8b980-8898-11ea-9b45-a2384765f408.png)

Where the expected result is 1, 2, r1.1, r2.1, 3, 4, etc in the column A. 
It looks like the method overwrites the cell with value "3".

Reply:
Sorry I did not test thoroughly. I fixed it but need you test again

Reply:
No problem, thank you for fast response